modules/reindex/src/test/java/org/elasticsearch/index/reindex/ReindexSourceTargetValidationTests.java

```diff
@@ -46,13 +46,15 @@
  */
 public class ReindexSourceTargetValidationTests extends ESTestCase {
```

Comment:
This class needs more work, but I am not sure whether we want reindex target-aliases to resolve to 
the write index. It feels right, but it also sounds scary. what do you think?

Reply:
oh nevermind, it *must* check details since even a single index with `is_write_index=false` should not be chosen by reindex api